public/js/scripts.js

```diff
@@ -604,23 +605,16 @@ function vote(type, id, dir) {
     }, 'json');
 }
 
-//submits a new question to store in the database, and then refreshes the questions
-//  with a different filter
-function submitQuestion() {
-    var text = $("#newQuestionTextarea").val();
```

Comment:
It was a duplicate. For some reason there were 2 submit question functions in the code. Maybe a merge thing that git hub missed? I do not know.

See line 618